.github/workflows/api-docs.yml

```diff
@@ -20,7 +26,7 @@ jobs:
       - name: Install dependencies
         run: yarn --prefer-offline --frozen-lockfile
       - name: Run typedoc
-        run: yarn build:docs
+        run: jq ".version" lerna.json | xargs yarn typedoc . --rootPackageVersion
```

Comment:
Can I ask why `documentVersion` and not use `version` in the `package.json` in the root? It seems the `package.json` in the root has a `version=0.0.0` and I am not sure why exactly. 

IMO reason why I made this change to read from `lerna.json`
- our monorepo seems to operate in ["Fixed/Locked mode"](https://lerna.js.org/docs/features/version-and-publish#fixedlocked-mode-default) so the version property in the root shold be the same as all the subpackages
- Initially, I wanted to use `version` in `package.json`, but there doesn't seem to be lifecycle hook such as `postversion` that operate at the root of the monorepo but even if there was I'd just be hooking a script that copies the version from `lerna.json`.

Reply:
I thought there was a specific reason for not using package.json version so I was suggesting adding a new property, but now I realize there is a specific reason to not use a package json that is not at the root. The rules around having a mono repo are that the root is actually not published, and is considered a private repo (a private repo is allowed to contain public repos). Since the root is not published it’s also not versioned. I see now why you are using the lerna version, because we are actually using lerna to drive the versioning of the mono repo. I forgot that was how our mono repo’s version worked. I think that make sense then that we are using lerna’s version since all the published packages in our mono repo derive its version from lerna’s version.

Reply:
For future reference, this plugin doesn't support independent versions of subpackages. This is ideal but requires a new feature implementation. The way typedoc works it can support this